install/kubernetes/helm/istio/charts/mixer/values.yaml

```diff
@@ -3,8 +3,6 @@
 #
 enabled: true
 replicaCount: 1
-autoscaleMin: 1
-autoscaleMax: 5
```

Comment:
These look to be used:
https://github.com/istio/istio/blob/fb756bc3a90d29270c0560497abf92becd9aac5d/install/kubernetes/helm/istio/charts/mixer/templates/autoscale.yaml#L10

Reply:
`$spec` is referring to either `istio-policy` or `istio-telemetry` and taking the values from those sections. Therefore those in the root are not being used and confuse as you just experienced.